### PR TITLE
Update to final nanoAOD version for Run 3

### PR DESCRIPTION
**Pull request by annamasce**

### CODE REVIEW

**Comment by steggema** [comment]
This looks good to me - @kandrosov should also have a look

File: NanoProd/config/skim_mu.yaml [hunk 1]
```diff
@@ -65,7 +65,6 @@ selection: "
                           && abs(Electron_dz) < 0.2 && abs(Electron_dxy) < 0.045
                           && (Electron_mvaIso_WP90 || (Electron_mvaNoIso_WP90 && Electron_pfRelIso03_all < 0.5));
   auto prompt_muons = Muon_pt > 20 && abs(Muon_eta) < 2.4
-                      && abs(Muon_dxy) < 0.045
```
Comment (kandrosov) [inline NanoProd/config/skim_mu.yaml:68]: What is the reason to remove dz and dxy for the prompt muon? Anyhow, if you decide to remove them, please do the same for prompt electrons to keep selections coherent.

File: NanoProd/python/DiMuon_cff.py [hunk 2]
```diff
@@ -235,25 +238,53 @@ def defineFiltersAndProducers(isRun2):
...
   this.dsaPatTable.variables.timeNdof = Var('time().nDof', float, doc='time ndof')
 
+  # track kink
+  this.dsaPatTable.variables.trkKink = Var('combinedQuality().trkKink', float, doc='trkKink')
+
+  # For some variables, save also bestTrack information
+  common_vars = ['chi2', 'ndof', 'dxy', 'dxyErr', 'dz', 'dzErr', 'pt_error', 'n_valid_hits', 'n_lost_hits']
+  for var_name in common_vars:
+    orig_var = getattr(this.dsaPatTable.variables, var_name)
+    final_expr = ''.join(['bestTrack.'] + orig_var.expr.value().split('.')[1:])
```
Comment (kandrosov) [inline NanoProd/python/DiMuon_cff.py:285]: Do we want to store some indicator for the type of best track?

File: NanoProd/config/skim_mu.yaml (hunk 1 above)
Comment (annamasce) [reply NanoProd/config/skim_mu.yaml:68]: Removing these two cuts allows for additional studies on their effect on signal efficiency, if requested by MuonPOG for SF validation. For consistency, they are removed also for prompt electrons in the following commit.

File: NanoProd/python/DiMuon_cff.py (hunk 2 above)
Comment (annamasce) [reply NanoProd/python/DiMuon_cff.py:285]: I already produced the data samples without this info, so I think it's fine for the moment.

Comment (steggema) [reply NanoProd/python/DiMuon_cff.py:285]: agreed, to be kept in mind for a possible future update

**Comment by steggema** [comment]
Merging since Konstantin's comments were addressed.

**Pull request merged**